src/ansys/simai/core/api/project.py

```diff
@@ -66,3 +66,19 @@ def set_project_sample(self, project_id: str, training_data_id: str):
 
     def delete_project(self, project_id: str):
         self._delete(f"projects/{project_id}", return_json=False)
+
+    def launch_build(
+        self,
+        project_id: str,
+        build_config: Dict[str, Any],
+        dismiss_data_with_fields_discrepancie: bool = False,
```

Comment:
yes, `Directory` has an abstract method `get` that has to have an implementation in the child classes. Would it make sense to let `pass` instead of raising an error?